src/goto-checker/properties.cpp

```diff
@@ -63,10 +63,17 @@ property_infot::property_infot(
 {
 }
 
-/// Return the properties in the goto model and initialize them to NOT_CHECKED
 propertiest initialize_properties(const abstract_goto_modelt &goto_model)
 {
   propertiest properties;
+  update_properties_from_goto_model(properties, goto_model);
```

Comment:
@tautschnig I believe this might be the resolution to your complain about check-properties code being deleted? (perhaps these two commits should be merged?)

Reply:
Where you said "incremental_goto_checker is already doing that."

Reply:
Ah... ok